src/context.rs

```diff
@@ -118,8 +129,10 @@ pub struct AppState {
     pub taffy: taffy::Taffy,
     pub(crate) view_states: HashMap<Id, ViewState>,
     pub(crate) disabled: HashSet<Id>,
+    pub(crate) keyboard_navigatable: HashSet<Id>,
     pub(crate) hovered: HashSet<Id>,
     pub(crate) cursor: CursorStyle,
+    pub(crate) using_keyboard_navigation: bool,
```

Comment:
Maybe you can name these `using_kbd_nav` and `kbd_navigatable`  to keep them a bit shorter, but its totally optional